package.json

```
@@ -133,13 +135,23 @@
     "clean": "./bin/clean.sh",
     "cover": "yarn run cover:actual; yarn run cover:collect",
     "cover:actual": "rm -rf .nyc_output/ coverage/ && lerna run cover --no-bail --concurrency $(node -p \"require(\\\"os\\\").cpus().length\")",
-    "cover:collect": "mkdir -p .nyc_output/ coverage/ && lerna exec -- \"if [ -d \"./.nyc_output\" ]; then cp -r .nyc_output ../..; fi;\" && nyc report --reporter=lcov",
+    "cover:actual:collect:lcov": "mkdir -p .nyc_output/ coverage/ && lerna exec -- \"if [ -d \"./.nyc_output\" ]; then cp -r .nyc_output ../..; fi;\" && nyc report --reporter=lcov",
+    "cover:collect": "yarn run test:collect; yarn run cover:actual:collect:lcov",
     "coveralls": "cat coverage/lcov.info | coveralls",
+    "check-run-reporter:actual": "curl \"https://api.check-run-reporter.com/api/v1/submissions\" --user $CHECK_RUN_REPORTER_TOKEN -X POST",
+    "check-run-reporter:test": "yarn run check-run-reporter:actual -- -F report=@\"./test-results.xml\" -F label=\"test\" -F sha=\"COMMIT_SHA\" -F root=\"$(pwd)\"",
```

Hi! I'm the CEO over at Check Run Reporter and I noticed some errors in the logs which are preventing you from seeing test results. I'll have a few documentation and code fixes coming along soonish, but in the meantime, you'll want to use the travis commit sha variable here:

```suggestion
    "check-run-reporter:test": "yarn run check-run-reporter:actual -- -F report=@\"./test-results.xml\" -F label=\"test\" -F sha=\"$TRAVIS_COMMIT\" -F root=\"$(pwd)\"",
```

I'm super excited to see folks using CRR so if you run into any other issues, feel free to reach out and I'll do what I can. Hope this helps!